kratos/includes/variables.h

```diff
@@ -126,6 +126,8 @@ namespace Kratos
 
     KRATOS_DEFINE_VARIABLE( double, TIME_INTEGRATION_THETA )
 
```

Comment:
Can you please put this to the ConDiffApp if it is only used there?

Reply:
Yes, for now it is, I moved it.